modules/gapi/perf/cpu/gapi_core_perf_tests_fluid.cpp

```diff
@@ -284,32 +332,36 @@ INSTANTIATE_TEST_CASE_P(ConvertToPerfTestFluid, ConvertToPerfTest,
 
 INSTANTIATE_TEST_CASE_P(ResizePerfTestFluid, ResizePerfTest,
     Combine(Values(Tolerance_FloatRel_IntAbs(1e-5, 1).to_compare_f()),
-        Values(CV_8UC3),
-        Values(cv::INTER_LINEAR),
-        Values(szSmall128, szVGA, sz720p, sz1080p),
-        Values(cv::Size(64, 64),
-               cv::Size(30, 30)),
-        Values(cv::compile_args(CORE_FLUID))));
+            Values(CV_8UC3),
+            Values(cv::INTER_LINEAR),
+            Values(szSmall128, szVGA, sz720p, sz1080p),
+            Values(cv::Size(64, 64),
+                   cv::Size(30, 30)),
+            Values(cv::compile_args(CORE_FLUID))));
 
 #define IMGPROC_FLUID cv::gapi::imgproc::fluid::kernels()
 INSTANTIATE_TEST_CASE_P(BottleneckKernelsPerfTestFluid, BottleneckKernelsConstInputPerfTest,
     Combine(Values(AbsSimilarPoints(0, 1).to_compare_f()),
-        Values("cv/optflow/frames/1080p_00.png", "cv/optflow/frames/720p_00.png",
-               "cv/optflow/frames/VGA_00.png", "cv/dnn_face/recognition/Aaron_Tippin_0001.jpg"),
-        Values(cv::compile_args(CORE_FLUID, IMGPROC_FLUID))));
+            Values("cv/optflow/frames/1080p_00.png", "cv/optflow/frames/720p_00.png",
+                   "cv/optflow/frames/VGA_00.png", "cv/dnn_face/recognition/Aaron_Tippin_0001.jpg"),
+            Values(0.5),
+            Values(0.5),
```

Comment:
Please remove this configuration parameters. fx and fy must be equals to 0.5 constantly. It shouldn't be possible to change them.

Reply:
Applied